foundation/www/docs/user/manual/en/accounts/index.txt

```diff
@@ -4,8 +4,10 @@ sales-invoice
 point-of-sale-pos-invoice
 point-of-sales
 purchase-invoice
+inter-company-invoices
```

Comment:
Restored docs via: #218. The PR bot was inactive for a couple of months, so we had missed updates here. Can you verify and update with the already present files?